linode/user/resource_test.go

```diff
@@ -119,8 +121,10 @@ func TestAccResourceUser_grants(t *testing.T) {
 					resource.TestCheckResourceAttr(testUserResName, "global_grants.0.add_linodes", "true"),
 					resource.TestCheckResourceAttr(testUserResName, "global_grants.0.add_longview", "false"),
 					resource.TestCheckResourceAttr(testUserResName, "global_grants.0.add_nodebalancers", "false"),
+					resource.TestCheckResourceAttr(testUserResName, "global_grants.0.add_placement_groups", "false"),
 					resource.TestCheckResourceAttr(testUserResName, "global_grants.0.add_stackscripts", "false"),
 					resource.TestCheckResourceAttr(testUserResName, "global_grants.0.add_volumes", "false"),
+					resource.TestCheckResourceAttr(testUserResName, "global_grants.0.add_vpcs", "false"),
```

Comment:
This file is what give me the most uncertainty. I confess to not digging to far into how it worked and just chose "false" in both cases.